vizro-core/docs/pages/user-guides/themes.md

```diff
@@ -101,5 +101,22 @@ px.scatter_matrix(
 )
 ```
 
+## Vizro themes in dash app (vizro-bootstrap)
+
+If you have a Dash app and want to use Vizro's themes, you can apply our `vizro.bootstrap` theme. This theme is visually similar to our `vizro_dark` and `vizro_light` themes but may have some differences because we use custom CSS to style certain Dash components that are not part of Bootstrap. You can view all our custom CSS files in our static folder [here](https://github.com/mckinsey/vizro/tree/main/vizro-core/src/vizro/static/css).
```

Comment:
```suggestion
If you have a Dash app and want to use Vizro's themes, you can apply the `vizro.bootstrap` theme. This theme is visually similar to our `vizro_dark` and `vizro_light` themes but has some differences because it uses custom CSS to style certain Dash components that are not part of Bootstrap. You can view all the custom CSS files in our static folder [here](https://github.com/mckinsey/vizro/tree/main/vizro-core/src/vizro/static/css).
```